.travis.yml

```diff
@@ -8,10 +8,33 @@ php:
   - 7.0
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
+        - php: 5.3
+          env: SYMFONY_VERSION=2.1.*
+    include:
+        - php: 5.3
+          env: SYMFONY_VERSION=2.1.*
```

Comment:
:+1: to remove it, but was just wondering or it actually still passes the tests because up to now we do support it according to `composer.json` at master.